pkg/haproxy/config.go

```diff
@@ -222,6 +236,83 @@ type RenderConfiguration struct {
 
 	// The load balancer port. Is not currently configurable.
 	Port uint32
+
+	// ExtraServices is the additional services that loadbalancer proxies to
+	ExtraServices []Frontend
+}
+
+// Frontend represent the loadbalancer endpoint that will  loadbalance the request to the backend servers
+type Frontend struct {
```

Comment:
This is confusing, since intuitively I am mapping `Frontend` to haproxy frontend, whereas this is more like the entire service object since it has the backend properties as well. I would instead rename `Frontend` to `Service` as it has both front end and back end attributes.

Reply:
Since haproxy configuration actually has backend reference in the frontend section, I kept it that way.
e.g.:
frontend svc_dex
default_backend "svc_dex_backend"

But, I agree for reader it might be confusing. Changed it to Service as per your recommendation